leakcanary-android-core/src/test/java/leakcanary/LeakCanaryConfigTest.kt

```diff
@@ -0,0 +1,51 @@
+package leakcanary
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import kotlin.reflect.full.memberFunctions
+import kotlin.reflect.full.memberProperties
+
+class LeakCanaryConfigTest {
+
+  private fun configBuilderFunctions() = LeakCanary.Config.Builder::class.memberFunctions
+      .map { it.name }
+
+  private fun configProperties() = LeakCanary.Config::class.memberProperties
```

Comment:
private helper functions should be after the tests (code should read outside in)